AndersenHWs/src/main/java/org/example/DAO/TicketDAO.java

```diff
@@ -0,0 +1,25 @@
+package org.example.DAO;
+
+import org.example.Ticket;
+import org.example.TicketType;
+
+import java.time.LocalDate;
+
+public class TicketDAO {
```

Comment:
please, move here logic from JDDC class
all operations related to connections and executing. and just call it in main class